lib/page-object/platforms/selenium_webdriver/page_object.rb

```diff
@@ -988,6 +988,22 @@ def elements_for(tag, identifier)
           find_selenium_elements(identifier, Elements::Element, tag.to_s)
         end
 
+        #
+        # platform method to return a PageObject rooted at an element
+        # See PageObject::Accessors#page_section
+        #
+        def page_for(identifier, page_class)
+          find_selenium_page(identifier, page_class)
+        end
+
+        #
+        # platform method to return a collection of PageObjects rooted at elements
+        # See PageObject::Accessors#page_sections
+        #
+        def pages_for(identifier, page_class)
+          SectionCollection.new(find_selenium_pages(identifier, page_class))
```

Comment:
Returns a collection of sections using a special class that provides extra searching niceties.